components/Movie.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import { useTheme, Caption, Headline, Divider, IconButton } from 'react-native-paper';
```

Comment:
when you are importing many things from a single library do not do it in a single line. It is easier to read and debug if they are in the column. 

``` jsx
import { 
  useTheme, 
  Caption, 
  Headline, 
  Divider, 
  IconButton } from 'react-native-paper'
```